docs/rewriter/rewrite_patterns.md

```diff
@@ -28,10 +28,10 @@ We will show how we can find a subgraph matching this computation and replace it
 Firstly, include all the rewriter relevant imports.
 
 ```python
-    from onnxscript.rewriter import pattern
-    from onnxscript import ir
+from onnxscript.rewriter import pattern
+from onnxscript import ir
 
-    _op = pattern.onnxop
+_op = pattern.onnxop
```

Comment:
I would check the code blocks below too. There are some more blocks that can be dedented